### ISSUE TITLE
Invalid map size when page was preloaded by browser

### ISSUE THREAD

**Issue by Zaczero** [openstreetmap-polska/openaedmap-frontend#225]
1. Enable preloading

![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/10835147/4091b0c8-5286-464d-ad5e-1563f434dd20)

2. Type in full address manually and wait few seconds

![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/10835147/90e8bcc9-7256-47ab-be25-2a9bb3b44cca)

3. Click enter

![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/10835147/afedb01d-5a41-456c-b50d-15400a636cce)

Basically it boils down to the invalid viewport size detection during preload (since there is no real viewport).
